packages/theia-integration/package.json

```diff
@@ -44,17 +44,17 @@
     "watch": "tsc -w"
   },
   "dependencies": {
-    "@eclipse-glsp/client": "2.1.0",
+    "@eclipse-glsp/client": "next",
     "ws": "~8.11.0"
   },
   "devDependencies": {
     "@types/ws": "^8.5.4"
   },
   "peerDependencies": {
-    "@theia/core": "^1.39.0",
-    "@theia/filesystem": "^1.39.0",
-    "@theia/messages": "^1.39.0",
-    "@theia/monaco": "^1.39.0"
+    "@theia/core": "^1.45.0",
```

Comment:
The breaking change was introduced with 1.44.x so this should be "^1.44.0".
I will update this in #196 accordingly